locales/fr.json

```diff
@@ -50,7 +50,7 @@
   "caught this error": "caught this error",
   "Cannot read property 'match' of undefined": "Cannot read property 'match' of undefined",
   "Government of Canada": "Government of Canada",
-  "required": "required",
+  "required": "champs requis",
```

Comment:
The common translation for required on GoC sites is "obligatoire" 
https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-fr.html

Reply:
Thanks for that catch! Updated to `obligatoire`.